drivers/include/sx126x.h

```diff
@@ -32,6 +32,27 @@
 extern "C" {
 #endif
 
+/**
+ * @brief   Whether there's only one variant of this driver at compile time or
+ * not.
+ */
+#define SX126X_SINGLE ((            \
+                          IS_USED(MODULE_SX1261) \
+                        + IS_USED(MODULE_SX1262) \
+                        + IS_USED(MODULE_SX1268) \
+                        + IS_USED(MODULE_LLCC68) \
+                        ) == 1)
+
+/**
+ * @brief   Variant of the SX126x driver.
+ */
+typedef enum {
+    SX126X_SX1261,
+    SX126X_SX1262,
+    SX126X_SX1268,
+    SX126X_LLCC68,
```

Comment:
Just as a suggestion, have the prefix `SX126X_TYPE_`, similar to the regulator mode enums.

Reply:
sure!

Reply:
done